contracts/base/Dispatcher.sol

```diff
@@ -269,9 +271,40 @@ abstract contract Dispatcher is Payments, V2SwapRouter, V3SwapRouter, V4SwapRout
                     }
 
                     (success, output) = address(V3_POSITION_MANAGER).call(inputs);
+                } else if (command == Commands.V4_INITIALIZE_POOL) {
+                    bytes4 selector;
+                    assembly {
+                        selector := calldataload(inputs.offset)
+                    }
+                    if (selector != PoolInitializer.initializePool.selector) {
+                        revert InvalidAction(selector);
+                    }
+                    (success, output) = address(V4_POSITION_MANAGER).call(inputs);
```

Comment:
What is the need for the call to go to the position manager? Wouldnt it be better to just call the pool manager directly? and instead of checking the selector just call `PoolManager.initialize(params)`? Or is there a reason to do so via the position manager?